app/assets/javascripts/home.js

```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
```

Comment:
I think it's a bit weird that there are all these dummy css and js files laying around, especially since files in `app/assets/{javascripts,stylesheets}` are only managed by the rails asset pipeline and not gulp.